studio-common/dot-studio/hab_toolchain

```diff
@@ -76,12 +76,12 @@ function install() {
 
   if [[ "x$1" == "x" ]]; then
     # Trying to use the last package built
-    cd /src || return
-    # If there are no packages available, build one for me
-    [[ ! -f results/last_build.env ]] && build .
-    source results/last_build.env
-    eval "hab pkg $install_cmd results/$pkg_artifact >/dev/null"
-    cd ../ || return
+    ( cd /src || exit
```

Comment:
+1 using subshell's to isolation `cd` is something I'd like to see us do in more of our scripts.